src/daos_vol.c

```diff
@@ -426,6 +414,67 @@ H5Pset_fapl_daos(hid_t fapl_id, const uuid_t pool_uuid, const char *pool_grp)
     D_FUNC_LEAVE_API;
 } /* end H5Pset_fapl_daos() */
 
+herr_t
+H5daos_set_prop(hid_t fcpl_id, const char *prop_str)
+{
+    htri_t      is_fcpl;
+    herr_t      ret_value = SUCCEED;
+
+    H5_daos_inc_api_cnt();
+
+    /* Check arguments */
+    if(fcpl_id == H5P_DEFAULT)
+        D_GOTO_ERROR(H5E_PLIST, H5E_BADVALUE, FAIL, "can't set values in default property list");
+    if((is_fcpl = H5Pisa_class(fcpl_id, H5P_FILE_CREATE)) < 0)
+        D_GOTO_ERROR(H5E_ARGS, H5E_BADTYPE, FAIL, "couldn't determine property list class");
+    if(!is_fcpl)
+        D_GOTO_ERROR(H5E_ARGS, H5E_BADTYPE, FAIL, "not a file create property list");
+    if(prop_str == NULL)
+        D_GOTO_ERROR(H5E_ARGS, H5E_BADVALUE, FAIL, "not a valid DAOS property string");
+
+    if(fcpl_id == H5P_DEFAULT)
+        D_GOTO_ERROR(H5E_PLIST, H5E_BADVALUE, FAIL, "can't set values in default property list");
```

Comment:
Looks like this line was duplicated from above

Reply:
yes will fix